homeassistant/components/airos/config_flow.py

```diff
@@ -28,14 +34,16 @@
         vol.Required(CONF_HOST): str,
         vol.Required(CONF_USERNAME, default="ubnt"): str,
         vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_SSL, default=DEFAULT_SSL): bool,
+        vol.Required(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): bool,
     }
 )
 
 
 class AirOSConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a config flow for Ubiquiti airOS."""
 
-    VERSION = 1
+    VERSION = 2
```

Comment:
Don't bump major version for this. Just accept missing verify_ssl as True.

Reply:
Taking the high road when unnecessary, learned something again 👍
Adjusted in `__init__` accordingly (and removed applicable tests)